influxdb_iox_client/src/client/operations.rs

```diff
@@ -122,4 +140,48 @@ impl Client {
             })?
             .into_inner())
     }
+
+    /// Return the Client Operation
+    pub async fn client_operation(&mut self, id: usize) -> ClientOperation {
+        let operation = self.get_operation(id).await.expect("get operation failed");
+        ClientOperation::try_new(operation).unwrap()
```

Comment:
Should this function return `Result` too, and propagate errors it might encounter rather than panicking? So:

```suggestion
    pub async fn client_operation(&mut self, id: usize) -> Result<ClientOperation> {
        let operation = self.get_operation(id).await?;
        ClientOperation::try_new(operation)
```

Reply:
Ye, I think we should propagate the errors